Sources/TemplateKit/Tag/DateFormat.swift

```diff
@@ -4,8 +4,37 @@
 ///
 /// If no date format is supplied, a default will be used.
 public final class DateFormat: TagRenderer {
+    private let defaultDateFormatter: DateFormatter
+
+    private static let dateAndTimeFormatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
+        return dateFormatter
+    }()
+
+    private static let iso8601Formatter: DateFormatter = {
+        let dateFormatter = DateFormatter()
+        dateFormatter.calendar = Calendar(identifier: .iso8601)
+        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
+        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)
+        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
+        return dateFormatter
+    }()
+
     /// Creates a new `DateFormat` tag renderer.
-    public init() {}
+    public convenience init() {
+        self.init(defaultDateFormatter: DateFormat.dateAndTimeFormatter)
+    }
+
+    /// Creates a new `DateFormat` tag renderer.
+    /// - parameter defaultDateFormatter: The date formatter to use when the tag invocation
+    ///   does not specify a date format.
+    public init(defaultDateFormatter: DateFormatter) {
```

Comment:
This also probably needs to be an escaping closure that creates a `DateFormatter`

Reply:
See above.